### PR TITLE
Added whisper timeout reasons to the emote limit module

### LINKED ISSUE #863: Add "Whisper Timeout Reason" option to Emote Limit module
Add "Whisper Timeout Reason" option to the Emote Limit module like the one in the Maximum Message Length module. 
image

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [ ] Documentation in docs/ or install-docs/ was updated, if applicable

Fixes #863

### CODE REVIEW

File: pajbot/modules/emote_limit.py [hunk 1]
```diff
@@ -93,6 +102,11 @@ def on_message(self, source, message, emote_instances, msg_id, **rest):
 
         if len(emote_instances) > self.settings["max_emotes"]:
             self.delete_or_timeout(source, msg_id, self.settings["timeout_reason"])
+            if self.settings["moderation_action"] == "Timeout":
```
Comment (RAnders00) [inline pajbot/modules/emote_limit.py:105]: there should definitely be a checkbox toggle for enabling or disabling the whisper

File: pajbot/modules/emote_limit.py [hunk 2]
```diff
@@ -70,6 +70,15 @@ class EmoteLimitModule(BaseModule):
             default="Too many emotes in your message",
             constraints={},
         ),
+        ModuleSetting(
+            key="whisper_timeout_reason",
+            label="Whisper Timeout Reason | Available arguments: {timeout_duration}",
+            type="text",
+            required=False,
+            placeholder="",
+            default="You have been timed out for {timeout_duration} for posting too many emotes.",
```
Comment (RAnders00) [inline pajbot/modules/emote_limit.py:79]: If anything, you missed a "seconds" here

```suggestion
            default="You have been timed out for {timeout_duration} seconds for posting too many emotes.",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Minor: Added whisper timeout reasons to the emote limit module
```
Comment (RAnders00) [inline CHANGELOG.md:5]:
```suggestion
- Minor: Added whisper timeout reasons to the emote limit module (#866)
```

File: pajbot/modules/emote_limit.py (hunk 2 above)
Comment (alazymeme) [reply pajbot/modules/emote_limit.py:79]: Huh? I thought I added that

File: pajbot/modules/emote_limit.py [hunk 4]
```diff
@@ -93,6 +109,14 @@ def on_message(self, source, message, emote_instances, msg_id, **rest):
 
         if len(emote_instances) > self.settings["max_emotes"]:
             self.delete_or_timeout(source, msg_id, self.settings["timeout_reason"])
+            if self.settings["moderation_action"] == "Timeout":
+                if self.settings["enable_whisper_timeout_reasons"] is True:
```
Comment (RAnders00) [inline pajbot/modules/emote_limit.py:113]: combine these two nested if-blocks with an `and` operator

**Pull request merged**